tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -98,6 +97,36 @@ export class QueryParamsFeatureFlagDataSource
     return featureFlags;
   }
 
+  persistFeatureFlags(flags: Partial<FeatureFlags>) {
+    const currentState = this.getPersistentFeatureFlags();
+    const newState = {
+      ...currentState,
+      ...flags,
+    };
+    localStorage.setItem(FEATURE_FLAG_STORAGE_KEY, JSON.stringify(newState));
+  }
+
+  resetPersistentFeatureFlag<K extends keyof FeatureFlags>(featureFlag: K) {
```

Comment:
nit: persistent seems to denote a subtype of feature flag rather than an action that has been performed on one. Maybe consider renaming to `resetPersistedFeatureFlag`

Reply:
You say you changed this but it has not been changed?